flink/v1.13/flink/src/test/java/org/apache/iceberg/flink/actions/TestRewriteDataFilesAction.java

```diff
@@ -331,7 +332,7 @@ public void testRewriteAvoidRepeateCompress() throws IOException {
     try (FileAppender<Record> fileAppender = genericAppenderFactory.newAppender(Files.localOutput(file), format)) {
       long filesize = 20000;
       for (; fileAppender.length() < filesize; count++) {
-        Record record = SimpleDataUtil.createRecord(count, "iceberg");
```

Comment:
Do we still need to change this line after we add the compress ratio according to this discussion ? https://lists.apache.org/thread/q9w7rwr31y4lykdkfx7dpdthk2x6bow1

Reply:
When I use the following formula:
`(long) (dataLength + (estimateMemory + (long) batch.size * avgRowByteSize) * Rate);`
If we use the "iceberg" as a test data, it is possible to pass this use case when the rate is less than or equal to 0.017. This value seems to be small.
A large number of repeated data, resulting in only 404 bytes of the generated files, and uuid-generated files about 22,2432 bytes.